service/src/main/kotlin/app/cash/backfila/service/persistence/DbRegisteredBackfill.kt

```diff
@@ -66,13 +66,17 @@ class DbRegisteredBackfill() : DbUnsharded<DbRegisteredBackfill>, DbTimestampedE
   @Column
   var requires_approval: Boolean = false
 
+  @Column
+  var delete_by: Instant? = null
```

Comment:
A similar comment here would be useful to document that this is just an advisory / notification, so the reader does not think this will cause something to be deleted

Reply:
Done